partials/tabs/projects.html

```diff
@@ -91,10 +91,11 @@
 
 <script>
   $(document).ready(function(){
-    $('.explore').click(function () {
-      $('html,body').animate({
-        scrollTop: $(".showcase").offset().top},
-        'slow');
+    $('ul.tabs').tabs();
+      $('.explore').click(function () {
```

Comment:
Why extra 2 spaces indentation here?

Reply:
that means travis is failing due to indentation  problem @li-boxuan ?